packages/fxa-auth-server/docs/swagger/subscriptions-api.ts

```diff
@@ -169,73 +178,123 @@ const OAUTH_MOZILLA_SUBSCRIPTIONS_CUSTOMER_PLAN_ELIGIBILITY = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/invoice/preview-subsequent',
-  notes: ['🔒 Authenticated with OAuth bearer token'],
+  notes: [
+    dedent`
+      🔒 Authenticated with OAuth bearer token
+
+      Preview a list of subsequent invoices.
+    `,
+  ],
 };
 
 const OAUTH_SUBSCRIPTIONS_COUPON_POST = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/coupon',
+  notes: ['Retrieves coupon details of a valid plan and promotion code.'],
 };
 
 const OAUTH_SUBSCRIPTIONS_PAYMENTMETHOD_FAILED_DETACH_POST = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/paymentmethod/failed/detach',
-  notes: ['🔒 Authenticated with OAuth bearer token'],
+  notes: [
+    dedent`
+      🔒 Authenticated with OAuth bearer token
+
+      Detaches a payment method from a customer without any subscriptions.
+    `,
+  ],
 };
 
 const OAUTH_SUBSCRIPTIONS_PRODUCTNAME_GET = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/productname',
+  notes: ['Returns the product name of a valid product id.'],
 };
 
 const OAUTH_SUBSCRIPTIONS_ACTIVE_SUBSCRIPTIONID_PUT = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/active/{subscriptionId}',
-  notes: ['🔒 Authenticated with OAuth bearer token'],
+  notes: [
+    dedent`
+      🔒 Authenticated with OAuth bearer token
+
+      Updates an active subscription for the user.
+    `,
+  ],
 };
 
 const OAUTH_SUBSCRIPTIONS_REACTIVATE_POST = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/reactivate',
-  notes: ['🔒 Authenticated with OAuth bearer token'],
+  notes: [
+    dedent`
+      🔒 Authenticated with OAuth bearer token
+
+      Reactivate valid customer subscription.
+    `,
+  ],
 };
 
 const OAUTH_SUBSCRIPTIONS_ACTIVE_NEW_PAYPAL_POST = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/active/new-paypal',
+  notes: ['Create subscription for the provided customer using PayPal.'],
 };
 
 const OAUTH_SUBSCRIPTIONS_PAYMENTMETHOD_BILLING_AGREEMENT_POST = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/paymentmethod/billing-agreement',
-  notes: ['🔒 Authenticated with OAuth bearer token'],
+  notes: [
+    dedent`
+      🔒 Authenticated with OAuth bearer token
+
+      Updates the billing agreement for a user with a new PayPal token.
+    `,
+  ],
 };
 
 const OAUTH_SUBSCRIPTIONS_STRIPE_EVENT_POST = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/stripe/event',
+  notes: [
+    'Handles webhook events from Stripe by pre-processing the incoming event and dispatching to the appropriate sub-handler.',
+  ],
 };
 
 const OAUTH_SUPPORTPANEL_SUBSCRIPTIONS_GET = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/support-panel/subscriptions',
-  notes: ['🔒 Authenticated with support panel secret'],
+  notes: [
+    dedent`
+      🔒 Authenticated with support panel secret
+
+      Returns all active support panel subscriptions for the given user id.
```

Comment:
Can probably list this as deprecated; see https://mozilla-hub.atlassian.net/browse/FXA-5543.

Reply:
Hm I don't see this listed as deprecated? Also the phrasing here is a little confusing. It makes it sound like there are support panel subscriptions, when these are really just our standard subscriptions (web and IAP...).

Reply:
Ahh thanks! I forgot to replace the notes with `This endpoint is deprecated.` and had only added the following lines to have the `Deprecated` flag next to the endpoint (see example below):

```
plugins: {
    'hapi-swagger': {
      deprecated: true,
    },
  },
```
<img width="560" alt="Screenshot 2023-02-01 at 9 12 29 AM" src="https://user-images.githubusercontent.com/28129806/216066389-02f666e6-86d2-404d-ad48-79d817ee8c61.png">

I'll update the notes section, thanks again!